docs/azure.rst

```diff
@@ -353,6 +353,11 @@ The following configuration options are available:
 ============================================== =================
 Name                                           Description
 ============================================== =================
+azure.azcopy.blockSize                          The block size (in MiB) when uploading to Azure Storage, and downloading from Azure Storage (default: , requires ``nf-azure@0.13.1``).
+azure.azcopy.blobTier                           The blob tier to be used while uploading to Azure Storage (default: , requires ``nf-azure@0.13.1``).
+azure.azcopy.putMD5                             Create an MD5 hash of each file during upload, and save the md5sum hash as the Content-MD5 property of the destination blob or file. (default: false)
+azure.azcopy.checkMD5                           Specifies how strictly MD5 hashes should be validated when downloading data. (default 'FailIfDifferent')
+azure.azcopy.overwrite                          Overwrite the conflicting files and blobs at the destination if this flag is set to true. (default: "false")
```

Comment:
is it really needed to give access to this attribute? the task should decide if overwrite the files or not

Reply:
The use-case I had in mind was that if a user completes the analysis and the results are uploaded (without the MD5 sum) - they could use this parameter to force the uploads with checksums.

> the task should decide if overwrite the files or not

Paolo, do you mean the `process.publishDir` mechanism here?

Reply:
No, I mean the md5 checksum can be turned on for any uploaded. I don't see any value giving it as an option to the user

Reply:
Paolo, circling back on this one - could you please confirm whether the option in question is `putMD5` or `checkMD5`? 

I'm thinking `putMD5` right? If so, I can accommodate this and make a push.

Reply:
Actually, both of them, would not be better just to enable by defualt

Reply:
Sure Paolo, I have accommodated this feedback and made the push.